damus/Views/Images/ImageContainerView.swift

```diff
@@ -35,10 +36,18 @@ struct ImageContainerView: View {
             }
             .imageModifier(ImageHandler(handler: $image))
             .clipped()
-            .modifier(ImageContextMenuModifier(url: url, image: image, showShareSheet: $showShareSheet))
+            .modifier(ImageContextMenuModifier(url: url, image: image, imageSaver: imageSaver, showShareSheet: $showShareSheet))
             .sheet(isPresented: $showShareSheet) {
                 ShareSheet(activityItems: [url])
             }
+            .alert(
+                "Failed to save photo",
+                isPresented: $imageSaver.error,
+                actions: {},
+                message: {
+                    Text("Please check the photo permissions in your Settings")
```

Comment:
I would clarify where the settings are: Settings.app. And since this is a complete sentence, add a period at the end.
```swift
Text("Please check the photo permissions for Damus in Settings.app.")
```